libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -78,6 +83,16 @@ export interface BaIndexPageContent {
   cta: BaStrapiCTA;
 }
 
+/** Icon page content */
+export interface BaIconPageContent {
```

Comment:
We now got properties like title, layout, ... in multiple interfaces. Let's create one and all content pages should extend from it.